build.gradle

```diff
@@ -30,4 +30,31 @@ test.finalizedBy("checkstyleMain")
 task runHelloWorldJoseG (dependsOn: 'classes', type: JavaExec) {
     main = 'jalau.at18.katas.helloworld.joseg.Main'
     classpath = sourceSets.main.runtimeClasspath
-}
\ No newline at end of file
+     standardInput = System.in
```

Comment:
Hi @saraialvarezrosas , following up on my previous comment, I think we can remove this line as it is not needed for this task